payments-ui-core/api/payments-ui-core.api

```diff
@@ -95,7 +99,7 @@ public final class com/stripe/android/ui/core/elements/AddressType$Normal : com/
 	public fun toString ()Ljava/lang/String;
 }
 
-public final class com/stripe/android/ui/core/elements/AddressType$ShippingCondensed : com/stripe/android/ui/core/elements/AddressType {
+public final class com/stripe/android/ui/core/elements/AddressType$ShippingCondensed : com/stripe/android/ui/core/elements/AddressType, com/stripe/android/ui/core/elements/AddressType$AutocompleteCapable {
```

Comment:
This seems like it shouldn't exist in this file, and shouldn't be considered public. Am I missing something? It's marked with restrict to in the code.

Reply:
Do we need to annotate all subclasses too?